packer/systemd/flask-app.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=flask-app service
+After=network.target
+
+[Service]
+ExecStartPre=/etc/flask-app/flask-app.sh
```

Comment:
Nice solution, but storing scripts in `/ets` isn't great idea.
`/usr/local/bin` is more preferable place